src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const everyDayPrice = 40;
+  const threeDayGet = 20;
+  const sevenDayGet = 50;
+
+  if ((days === '') || (days === 0)) {
+    return 0;
+  }
+
+  if (days >= 7) {
+    return (days * everyDayPrice - sevenDayGet);
```

Comment:
Good job!
make a separate variable for "days * everyDayPrice", you used it three times